app/assets/javascripts/application.js

```diff
@@ -54,6 +44,16 @@ function onContentLoad() {
   tfm.tools.activateTooltips();
   tfm.tools.activateDatatables();
 
+  // Prevents all links with the disabled attribute set to "disabled"
+  // from being clicked.
+  $('a[disabled="disabled"]').click(function(evt) {
```

Comment:
This works up until the point jquery rerenders some button and do not call `onContentLoad`.
Have you tested disabled buttons while the task is in progress? The toolbar gets rerendered every few seconds and the `onContentLoad` is not called I believe, that's why we moved the handler on `document`.

Reply:
Ah, you're right. It works on a job that is already done, but is broken while it is still running.

Reply:
updated, calling that method also outside the `contentLoad`